### DISCUSSION TITLE
隐藏首选项+候选唯一时隐藏候选

### DISCUSSION THREAD

**Discussion by AlseinX** [rime/home#915] [Ideas]
求支持隐藏首选项和候选唯一时隐藏候选的功能
因为首选可以设置为嵌入到输入的地方，所以不需要在候选框中显示首选项
一些可以盲打的方案，很多时候都是有唯一候选的
这样就很傻
![image](https://user-images.githubusercontent.com/8338587/140497500-fce57d8a-66bb-4efc-a57d-1b864076bcc1.png)
![image](https://user-images.githubusercontent.com/8338587/140497545-a4263da1-7ca8-4b7e-abc2-b9d43408b853.png)

**Comment by lotem** [comment]
有趣的提議

**Comment by MokOopsing** [comment]
候选栏倒是有办法隐藏

![image](https://user-images.githubusercontent.com/39441028/144066620-b1c82393-7f94-4596-a5ea-5858cbe31b9f.png)


但是没法动态隐藏

**Comment by KeithW0726** [comment]
现在是24年的下半年了，还能实现这个功能不，挺不错的

**Comment by xbeta-zhang** [comment]
试用冰凌五笔的「默认隐藏候选框+4码有重码时显示候选框」，想查一下rime是否有更灵活的方案，搜索到这里。

**Comment by fxliang** [comment]
有啊，实现不太难。想好完整的隐藏条件，尽量满足多的潜在需求

**Comment by MokOopsing** [comment]
https://github.com/hchunhui/librime-lua/issues/204  没准可以参考这个用lua实现
